.github/ISSUE_TEMPLATE/issue.yml

```
@@ -0,0 +1,27 @@
+  - type: textarea
+    attributes:
+      label: 📄 Description
+      description: 이슈에 대한 설명을 작성해 주세요.
+      placeholder: 구체적인 상황과 배경을 포함해서 작성해주세요.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: ✅ Tasks
+      description: 이슈 해결을 위한 세부 작업 목록을 체크리스트로 작성해주세요.
+      placeholder: |- 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)